department-of-veterans-affairs/va.gov-cms
# VACMS-6158 Update flag styling

## Description
Updated colors to match design recommendation. 

Relates to #6518 .

## Testing done


## Screenshots


## QA steps

As cms user
1. Go to /admin/content/flagged and filter
2. See flags that are styled instead of plain text


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
